securedrop/sass/_base.sass

```diff
@@ -98,14 +102,8 @@
   +aside
   +flash
   +banner-warning
-
-  div
-    &#select-container
-      display: inline
-
-    &.document-actions button
-      font-size: 14px
-      padding: 6px 15px
+  +select-container
+  +document-actions-button
```

Comment:
These are not an exact copy of the original. They were both under the same **div** but this factorization did not make sense from a functional point of view and there is no harm in breaking it.